### PR TITLE
Convert TemplateList to PaginatedTable

### PR DESCRIPTION
**Pull request by keithjgrant**
##### SUMMARY
Converts TemplateList component to use PaginatedTable. I also added the expandable content rows

addresses #6189

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
 - UI
 - 
Screen Shot 2021-01-19 at 2 59 47 PM

### CODE REVIEW

**Review by jakemcdermott — APPROVED** [review]

**Comment by tiagodread** [comment]
Awesome Job @keithjgrant, so far, I've found these items:

- Expanded view missing `Last modified` and `Last Ran`:

image

- Workflow Job Template doesn’t contain project, we are displaying on the Expanded view:

image
Old:
image

- Inventory list contains a column header for `Actions`:

image


But this was not applied for `Jobs list` AND `Job Template list`:

image
 
- The column Last Run doesn’t make sense when you try to sort IMO, because this is showing the playbook activity instead of the Last Run timestamp value, maybe if we rename this column to `Activity` (not sortable) and add a new column for `Last Run` with timestamp (sortable) would be better. cc @keithjgrant @trahman73 

image
 
- Add data-ouia-id (when available) or IDs on:

 - each sort option (name, type, Last run...)
 - each action button
 - each <dd> element on expanded view

**Comment by trahman73** [comment]
@tiagodread If it is not sortable, we can perhaps show Activity in the expanded area?

**Comment by tiagodread** [comment]
> @tiagodread If it is not sortable, we can perhaps show Activity in the expanded area?

Yeah @trahman73 we can do that, actually looks better than my suggestion 😄

**Comment by keithjgrant** [comment]
@tiagodread Just pushed fixes for the items you listed.

I also changed the Last Run column to display a timestamp of the last run, leaving the detail sparkview in the expanded column

**Comment by tiagodread** [comment]
recheck

**Comment by tiagodread** [comment]
@keithjgrant the Dashboard -> Recent Templates List don't inherit the `Templates` the same PaginatedTable list?

jt

**Review by jakemcdermott — APPROVED** [review]

File: awx/ui_next/src/components/DetailList/Detail.jsx [hunk 1]
```diff
@@ -34,6 +34,7 @@ const DetailValue = styled(
 `;
 
 const Detail = ({
+  id,
```
Comment (tiagodread) [inline awx/ui_next/src/components/DetailList/Detail.jsx:37]: @keithjgrant defined but never used

Comment (tiagodread) [inline awx/ui_next/src/components/DetailList/Detail.jsx]: @keithjgrant defined but never used

**Review by tiagodread — COMMENTED** [review]
only a small lint error, after fixing this, this is ready to go.

**Review by tiagodread — APPROVED** [review]
Nice! fixed broken tests here https://github.com/ansible/tower-qa/pull/5917

**Pull request merged**